Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -289,39 +289,39 @@ private void UploadDirectoryToGCSBucket(string directory, Dictionary<string, str
         {
             if (string.IsNullOrWhiteSpace(directory) || !Directory.Exists(directory))
             {
-                // do nothing if directory is not valid
+                // Do nothing if the directory is not valid.
                 return;
             }
 
-            // have to append slash to create directory on the cloud
+            // We have to append / at the end to create a directory on the cloud.
             string relativeDirectoryNameWithSlash = relativeDirectoryName + "/";
 
             if (TestObjectExists(Service, Bucket, relativeDirectoryNameWithSlash) && !Force.IsPresent)
             {
-                // if the directory exists and force is not present, throw an error
+                // If the directory exists and force is not present, throw an error.
                 throw new PSArgumentException(
                     $"Storage object '{relativeDirectoryNameWithSlash}' already exists. Use -Force to overwrite.");
             }
 
-            // create a directory on the cloud
+            // Create a directory on the cloud.
             string objContentType = GetContentType(null, metadataDict, UTF8TextMimeType);
             byte[] contentBuffer = Encoding.UTF8.GetBytes(Contents);
             Stream contentStream = new MemoryStream(contentBuffer);
             UploadStreamToGCSObject(contentStream, objContentType, metadataDict, relativeDirectoryNameWithSlash);
 
             foreach (string file in Directory.EnumerateFiles(directory))
             {
-                // upload each individual file
+                // Upload each individual file.
                 UploadStreamToGCSObject(new FileStream(file, FileMode.Open),
                     GetContentType(ContentType, metadataDict, InferContentType(file)),
                     metadataDict,
-                    // get a relative path compared to the root path, also have to replace \ with / so it will be created with correct folder structure
+                    // Get a relative path compared to the root path, also we have to replace \ with / so it will be created with correct folder structure.
                     Path.Combine(relativeDirectoryName, Path.GetFileName(file)).Replace('\\', '/'));
```

Comment:
Given that we call `.Replace('\\', '/')` in a couple of places, perhaps we can make a local function to make it clearer what is going on?

`private static string ConvertLocalToGcsFolderPath(string localFilePath) {`